README.md

```diff
@@ -1,42 +1,38 @@
 # ROCm for Arch Linux
 This repository hosts a collection of [Arch Linux](https://www.archlinux.org/) [PKGBUILDs](https://wiki.archlinux.org/index.php/PKGBUILD) for the [AMD ROCm Platform](https://rocmdocs.amd.com/en/latest/).
-These scripts implement a great portion of the stack, ranging from low level interfaces, over compilers and high-level application librariers.
+These scripts implement a great portion of the stack, ranging from low level interfaces, over compilers to high-level application librariers.
 
 ## Installation
 The Arch Linux packages for ROCm are available on the [AUR](https://wiki.archlinux.org/index.php/Arch_User_Repository).
-Since many packages will be installed, it is recommended to use an [AUR helper](https://wiki.archlinux.org/index.php/AUR_helpers)
-like [`yay`](https://aur.archlinux.org/packages/yay/).
-
-It is also recommended to use the [`arch4edu`](https://wiki.archlinux.org/index.php/Unofficial_user_repositories#arch4edu) binary repository as it will
+Since many packages will be installed, we recommend to use the [`arch4edu`](https://wiki.archlinux.org/index.php/Unofficial_user_repositories#arch4edu) binary repository as it will
```

Comment:
I think we should keep the `yay` instructions. We are recommending to use arch4edu repo, already and I think keeping yay covers packages that are not in arch4edu. Until all packages in this repo are in arch4edu I do not feel comfortable changing this.

Reply:
From a developer's standpoint, I like people that use yay or makepkg since that means more bugs get reported and issues found.

Reply:
You're right. The last point is very important as we've seen in #406